rmgpy/tools/regression.py

```diff
@@ -101,6 +101,15 @@ def read_input_file(path):
         new_imfs.append(new_dict)
     setups[3] = new_imfs
 
+    # TODO repeat this for surface species, make sure it's happening for them too
```

Comment:
Same as previous, please open or link to an issue on GitHub to track this TODO

Reply:
I forgot to take this TODO tag out. The next few lines of code implement what it's describing. I've updated the code to   reflect this.